images/base-notebook/Dockerfile

```
@@ -39,9 +39,14 @@ USER ${NB_UID}
 # files across image layers when the permissions change
 WORKDIR /tmp
 RUN mamba install --yes \
-    'jupyterhub' \
+    'jupyterhub-base' \
     'jupyterlab' \
     'nbclassic' \
+    # nodejs has historically been installed indirectly as a dependency. When it
+    # was no longer getting installed indirectly, we started installing it
+    # explicitly to avoid introducing a breaking change for users building on
+    # top of these images.
```

```suggestion
    # nodejs has historically been installed indirectly as a dependency.
    # When it was no longer getting installed indirectly,
    # we started installing it explicitly to avoid introducing a breaking change
    # for users building on top of these images.
    # See: https://github.com/jupyter/docker-stacks/pull/2171
```

We use slightly different style of comments in this repo